wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/document/DocumentDetailsScreen.kt

```diff
@@ -156,14 +161,34 @@ private fun DocumentAuthenticationRequired(
 private fun DocumentDetails(
     documentInfo: DocumentInfo,
 ) {
-    Column() {
-        Text(
-            text = stringResource(R.string.document_details_screen_flash_pass_lecture),
-            textAlign = TextAlign.Center,
-            fontStyle = FontStyle.Italic,
-            style = MaterialTheme.typography.bodySmall,
-            modifier = Modifier.padding(8.dp)
-        )
+    Column(
+        modifier = Modifier.padding(top = 20.dp)
+    ) {
+        Row(
+            modifier = Modifier
+                .padding(horizontal = 30.dp)
```

Comment:
nit: It looks like all the elements on the screen should be indented from the left so they line up with this box. Maybe move this from this Row to the Column that contiains everything?